로그인 UI Grid 적용

riverandeye opened the pull request:
로그인 UI에 Grid를 적용하여 리팩토링 하였습니다.

riverandeye committed:
[client] | refactor(auth): login page에 Grid 추가

코드의 일관성을 위해 Grid를 추가하였습니다.

riverandeye committed:
[client] | refactor(auth): signup page에 Grid 추가

코드의 일관성을 위해 Grid를 추가하였습니다.

Atercatus reviewed (changes requested):
혁우님이 다시 한번 확인 해달라고 하십니다. 월요일에 같이 말씀을 나눠봐요

hyokualexkwon commented on packages/client/views/SignUp/styles.tsx:
```diff
@@ -46,14 +43,14 @@ export const RequireMark = styled.div`
   flex-direction: column;
   align-items: flex-start;
   width: 100%;
+  max-width: 58.8rem;
```
여기에는 max-width를 주면 안되겠죠?

hyokualexkwon commented on packages/client/views/SignUp/styles.tsx:
```diff
@@ -34,9 +31,9 @@ export const Form = styled.div`
+  justify-content: center;
   width: 100%;
   vertical-align: top;
-  max-width: 58.8rem;
```
max-width 1200px을 줘야겠죠?

![image](https://user-images.githubusercontent.com/17868599/69925352-09aa8500-14f3-11ea-84df-d9ae8b83c99b.png)

hyokualexkwon commented on packages/client/views/SignUp/index.tsx:
```diff
@@ -35,52 +36,58 @@ const SignUp: React.FunctionComponent = () => {
-        </S.Form>
-      </S.Container>
+      <S.ContainerGrid container justify="center">
+        <Grid item xs={12} md={8}>
```
md 값이 8이 아니고 6이겠죠?

![image](https://user-images.githubusercontent.com/17868599/69925355-18913780-14f3-11ea-8d07-3c539c6fd029.png)

riverandeye committed:
[client] | refactor(SignUp): 가입 양식 페이지 PR 리뷰 반영

1. Padding을 ContainerGrid에서 적용하지 않고, 상위 Container로 옮겼습니다.
2. 버튼에 pointer 옵션을 추가하였습니다.
3. 잘못된 max-width 설정을 수정하였습니다.

KimJye commented on packages/client/views/Login/index.tsx:
```diff
@@ -1,23 +1,30 @@
+            커뮤니티에 가입하고, 여러분의 소중한 개발 경험을 공유해 주세요!
+          </S.Message>
+          <S.Link>
+            <S.AuthenticateLink href="#">
```
혹시 이부분 저희 아직 api안됐나요?

riverandeye replied:
앗 잘은모르겠어요.. 로그인 테스트 할 때 반영하면 좋을것 같아요!

KimJye commented on packages/client/views/SignUp/index.tsx:
```diff
@@ -36,50 +37,61 @@ const SignUp: React.FunctionComponent = () => {
+        <S.ContainerGrid container justify="center">
+          <Grid item xs={12} md={6}>
+            <S.HeadMessage>거의 다 되었어요!</S.HeadMessage>
+            <S.Form onSubmit={() => null}>
```
의미하는 바가 무엇인가요?? 신기해요~

Atercatus replied:
지워 주세요 ㅎㅎ

riverandeye replied:
머지할떄 반영하겠습니다.

hyokualexkwon replied:
12

KimJye reviewed (approved):
고생하셨습니다

Atercatus commented on packages/client/views/Login/index.tsx:
```diff
@@ -1,23 +1,30 @@
 import React from 'react';
 import * as S from './styles';
 import { LogoSVG, GitHubSVG } from '../../svgs';
+import Grid from '@material-ui/core/Grid';
```
import 라이브러리는 위로 올리고 코드는 아래로 내리고 사이에 빈 공백 넣어주세요

Atercatus commented on packages/client/views/Login/styles.tsx:
```diff
@@ -1,13 +1,13 @@
 import styled from 'styled-components';
 import { BREAKPOINT } from '../../constants';
+import MaterialGrid from '@material-ui/core/Grid';
```
여기도용

Atercatus commented on packages/client/views/SignUp/styles.tsx:
```diff
@@ -1,28 +1,29 @@
 import styled from 'styled-components';
 import { BREAKPOINT } from '../../constants';
+import MaterialGrid from '@material-ui/core/Grid';
```
여기도용

riverandeye committed:
[client] | refactor(auth): 로그인 및 회원가입 페이지 리팩토링

PR에 작성된 요구사항 반영하였습니다.

hyokualexkwon commented on packages/client/views/SignUp/styles.tsx:
```diff
@@ -1,28 +1,30 @@
   text-align: center;
   color: white;
-  font-size: 2.8rem;
+  font-size: 3.2rem;
```
모바일에서 2.8이고, 데스크톱에서 3.2네요! 둘 다 적용해야 할것 같은데요?

riverandeye replied:
반영하였습니다!

riverandeye committed:
[client] | refactor(auth): 로그인 및 회원가입 페이지 리팩토링

PR에 작성된 요구사항 반영하였습니다.

riverandeye committed:
Merge branch 'develop' into feature/login_ui_20191125
